plugins/modules/ipahbacrule.py

```diff
@@ -469,21 +469,36 @@ def main():
             elif state == "enabled":
                 if res_find is None:
                     ansible_module.fail_json(msg="No hbacrule '%s'" % name)
+                # FreeIPA 4.9.10+ and 4.10 use proper mapping for
+                # boolean values, so we need to convert it to str
+                # for comparison.
+                # See: https://github.com/freeipa/freeipa/pull/6294
+                is_enabled = (
+                    str(res_find["ipaenabledflag"][0]).upper() == "TRUE"
```

Comment:
Is "ipaenabledflag" always set in res_find?

Reply:
Fixed.